### DISCUSSION TITLE
Improve the README to serve the server locally, along with solutions for potential issues.

### DISCUSSION THREAD

**Discussion by PedrelliMath** [LBPUnion/ProjectLighthouse#1047] [Ideas]
I don’t understand anything about C# and docker compose up -d doesn’t make the server work locally; we need an improved README. I’m not sure if I can get this working on my computer just by reading the README. It would be helpful to have a complete tutorial for self-hosting, downloading and serving community maps, and so on.

**Comment by m88youngling** [comment]
I'm in a similar position as you. My goal is to try and getting it working on my system as well with the end goal of writing a tutorial on how to host Project Lighthouse for yourself. PR #1030 is something I realized I needed to do to get it working, but I'm stuck on a problem connecting the database. My PR changes the name of the image that's pulled in the docker compose file. Maybe that might help you make some more progress instead of having to build the whole project from source?
